Packs/GitHubITAdmin/Integrations/GitHub_IT_Admin/GitHub_IT_Admin.yml

```diff
@@ -0,0 +1,268 @@
+category: IT Services
+commonfields:
+  id: GitHub IT Admin
+  version: -1
+configuration:
+- defaultvalue: ""
+  display: GitHub URL https://<domain>.github.com/
+  name: url
+  required: true
+  type: 0
+- defaultvalue: ""
+  display: token
+  name: token
+  required: true
+  type: 4
+- defaultvalue: ""
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: ""
+  display: organization name
+  name: org
+  required: false
+  type: 0
+- additionalinfo: An optional custom mapping that takes custom values in the SCIM
+    data into the integration.
+  defaultvalue: ""
+  display: Custom Mapping for Create User
+  name: customMappingCreateUser
```

Comment:
```suggestion
  name: custom_mapping_create_user
```

Reply:
this is a part of design and consistent in all the integrations for User Provisioning